src/share/poudriere/common.sh

```diff
@@ -684,6 +684,7 @@ buildlog_start() {
 	echo "maintained by: $(injail /usr/bin/make -C ${portdir} maintainer)"
 	echo "Makefile ident: $(ident -q ${mnt}/${portdir}/Makefile|sed -n '2,2p')"
 	echo "Poudriere version: ${POUDRIERE_VERSION}"
+	[ ${QEMU_EMULATING} -eq 1 ] && echo "QEMU version: `pkg query %v qemu-user-static`"
```

Comment:
This should be `${PKG_BIN} query ...`. But more importantly we don't know what _qemu-user-static_ is what is being used. It could be any qemu package from any qemu origin.